modules/cluster-logging-updating-logging-to-5-0.adoc

```diff
@@ -49,11 +38,11 @@ Upgrade your cluster logging version to 4.6 before updating to Red Hat OpenShift
 
 .. Click *Subscription* -> *Channel*.
 
-.. In the *Change Subscription Update Channel* window, select *5.0* and click *Save*.
+.. In the *Change Subscription Update Channel* window, select *5.0* or *stable-5.1* and click *Save*.
```

Comment:
@anpingli or @kabirbhartiRH Would either of you verify this in the UI?

Reply:
@rolfedh The Change Subscription update channel currently shows stable or stable-5.1 in case you want to upgrade to 5.1. But it is correct for the case you want to move to either 5.0 or 5.1.